common/pom.xml

```diff
@@ -45,6 +45,7 @@
     <dependency>
       <groupId>org.apache.calcite</groupId>
       <artifactId>calcite-core</artifactId>
+      <version>${calcite.version}</version>
```

Comment:
Why is this change necessary? The version should come from `<dependencyManagement>` of the parent pom.

Reply:
Will it be better to explicitly specify the version of calcite-core in hive-exec?